docs/integrations/monitor-non-service-metrics-prometheus.rst

```diff
@@ -0,0 +1,101 @@
+Monitor non-service-specific metrics using Prometheus
+========================================================
+
+Prometheus integration allows you to monitor your Aiven services and understand the resource usage. Additionally, using this integration, you can monitor/track some non-service-specific metrics that may be worth monitoring. 
+
+To start monitoring these metrics, you need the :doc:`Prometheus integration configured and set up the Prometheus server <../platform/howto/integrations/prometheus-metrics>`.
+
+The complete list of metric you can monitor are not listed here. However, you can get the complete list in the Prometheus service or by sending ``HTTP GET`` with ``cURL`` as shown below: 
+
+.. code:: 
+  
+    curl --insecure --user promXXXX:XXXXXXXXXXXXXXXX https://YOUR-SERVICE-FQDN:9273/metrics
+
+
+CPU usage 
+----------
+CPU usage metrics are helpful in determining if the CPU is constantly being maxed out.
+For a high-level view of CPU usage for a single CPU service, you can use
+
+.. code:: 
+  
+    100 - cpu_usage_idle{cpu="cpu-total"}
+
+Bear in mind that it may not be a good idea to use ``cpu_usage_system{cpu="cpu-total"}`` + ``cpu_usage_user{cpu="cpu-total"}`` to calculate total CPU usage, because they do not include ``cpu_usage_iowait``. Furthermore, a process with a ``nice`` value larger than 0 will be categorized as ``cpu_usage_nice``, which is not included in ``cpu_usage_user``.
+
+It is also worth monitoring ``cpu_usage_iowait{cpu="cpu-total"}``. If this number is high, it indicates the service node is working on something I/O intensive. For example, 40 means 40% of CPU time is waiting for disk or network I/O.
+
+The following table lists some important CPU-related metrics you can collect and monitor:
+
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Metics
+    - Description
+  * - ``cpu_usage_idle``
+    - The percentage of time the CPU is ideal. 
+  * - ``cpu_usage_system``
+    - The percentage of time the Kernel code is consuming the CPU. 
+  * - ``cpu_usage_user``
+    - The percentage of time the CPU is in user-space program with nice value <= 0.
```

Comment:
I think that "nice" here is a technical term rather than an English word, we could make it code marked-up?

Reply:
fixed